Updating verbiage for Terraform Cloud Business Tier (#101)
hashicorp/terraform-docs-common: website/docs/docs/partnerships.mdx

Partners who successfully complete the Terraform Cloud integration checklist obtain a Terraform Cloud badge. This signifies HashiCorp has verified the integration and the partner is a member of the HashiCorp Technology Partner Program.

> - Note: Currently, pre-apply is the only integration phase available. As of September 2021, run tasks are available only as a beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization since this is currently enabled by default for our business tier customers of Terraform Cloud. If you have a customer that is interested in run tasks and are not a current Terraform Cloud Business Tier customer please request them to [sign up using the form](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).

Reviewer: ```suggestion
- Note: Currently, pre-apply is the only integration phase available. As of September 2021, run tasks are available only as a beta feature, are subject to change, and not all customers see this functionality in their Terraform Cloud organization since this is currently enabled by default for our business tier customers of Terraform Cloud. If you have a customer who wants to run tasks and is not a current Terraform Cloud Business Tier customer, please ask them to [sign up using the form](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).
```

Suggest changing "will see" to "see" to avoid using future tense. Suggest changing "customer that is interested in" to "customer who wants to" to avoid a passive voice style issue. Also suggest changing "are not a current" to "is not a current" to keep subject-verb agreement since it's only "a customer" rather than "several customers". Also suggest changing "request" to "ask" because "ask" is a shorter, less complicated word.
